ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -129,11 +130,13 @@ export function RecipientWithAddress({
     );
   }
 
+  const normalizedEnsRecipientName = toUnicode(recipientEns);
+
   const displayName =
     (recipientName ||
       recipientNickname ||
       recipientMetadataName ||
-      recipientEns ||
+      normalizedEnsRecipientName ||
```

Comment:
Will we have this issue rendering ENS proposed names in petnames, and anywhere the ENS controller state is used?

Could we fix this deeper in the `getEnsResolutionByAddress` selector perhaps so we don't have to manually do this for each usage?

Reply:
Make sense to do that casting on deeper, done.